src/components/spatial/Spatial.js

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
 
-import { ScatterplotLayer, PolygonLayer, COORDINATE_SYSTEM } from 'deck.gl';
-import { SelectablePolygonLayer } from '../../layers';
+import {
+  ScatterplotLayer, PolygonLayer, COORDINATE_SYSTEM, BitmapLayer,
+} from 'deck.gl';
+import { SelectablePolygonLayer, IdentityCoordinatesTileLayer } from '../../layers';
 import { cellLayerDefaultProps, PALETTE, DEFAULT_COLOR } from '../utils';
 import AbstractSelectableComponent from '../AbstractSelectableComponent';
 import LayersMenu from './LayersMenu';
-import OpenSeadragonComponent from '../../vendor/OpenSeadragonComponent';
 
 export function square(x, y, r) {
   return [[x, y + r], [x + r, y], [x, y - r], [x - r, y]];
 }
 
+function loadImage(src) {
+  // this function replaces load from loaders.gl which was not working as of
+  // this package version of deckgl (7.1.4)
```

Comment:
- Please capitalize, include period at end.
- More accurate to say we just couldn't get it working, and redoing it was simpler... unless there is a bug filed?

Reply:
I'll rephrase to make it clearer I personally could not get this version of `loaders.gl` working with this version of `deck.gl`